Source/WTF/wtf/LLVMProfileUtils.h

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2025  All rights reserved.
+
+#pragma once
+
+#if ENABLE(LLVM_PROFILE_GENERATION) && PLATFORM(IOS_FAMILY)
+#include "text/MakeString.h"
+#include "text/WTFString.h"
```

Comment:
```suggestion
#include <wtf/text/MakeString.h>
#include <wtf/text/WTFString.h>
```

…for consistency with other WTF code.